### ISSUE TITLE
Document that Glucose Safety Limit is bounded by Pre-Meal Range

### ISSUE THREAD

**Issue by whiten** [LoopKit/Loop#1592]
I upgraded from 2.2.6 to the dev branch at cf6e2f25 last evening, and went through the onboarding flow. My suspend limit coming from 2.2.6 was 65, and during the dev onboarding flow I adjusted this up slightly, maybe to 67.

Now that I'm in the app, when I try to update my Glucose Safety Limit (for which I'm presented a warning on the Therapy Settings page), the spinner only shows the values 67 to 70, and the Save button is disabled if 70 is selected. That is, I cannot move the value above 69.

![IMG_8850](https://user-images.githubusercontent.com/162028/150830614-ffa4f187-95d5-4db9-834c-23b7aeb71322.PNG)
![IMG_8851](https://user-images.githubusercontent.com/162028/150830631-5c96d76c-10aa-4882-964a-059ae8b2e0a7.PNG)

I'm on an iPhone 12 Pro, iOS 15.1.1. Loop reports "v2.3.0 (57)" as its version.

**Comment by ps2** [comment]
What is the low end of your target range? Safety limit cannot be set above that.

**Comment by whiten** [comment]
Correction range is 100-115, but my Pre-Meal Range is 70-90. When I moved the low end of my Pre-Meal Range up, I was able to raise this to the lower bound of that. I am not in a Pre-Meal Period right now.

Changed the title to correct that this is an in-app documentation issue and not a code bug (apart from the Save button being disabled at the 70 value here perhaps).

The Pre-Meal Range info page does say that it can be set as low as the Glucose Safety Limit, but the Glucose Safety Limit's info page doesn't have a similar reference to Pre-Meal Range. Adding such a reference is how I would improve this.
